styleguide/screens/Patterns/Inputs/Inputs.js

```diff
@@ -0,0 +1,417 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { H, T, D, C, A } from '../../../components/Scaffold/Scaffold';
+
+import CheckboxGroup from '../../../../components/Form/CheckboxGroup/CheckboxGroup';
+import IconInput from '../../../../components/Form/IconInput/IconInput';
+import Input from '../../../../components/Form/Input/Input';
+import InputRange from '../../../../components/Form/InputRange/InputRange';
+import InputRangeWithHistogram
+  from '../../../../components/Form/InputRange/InputRangeWithHistogram';
+import Option from '../../../../components/Form/Select/Option';
+import RadioGroup from '../../../../components/Form/RadioGroup/RadioGroup';
+import Select from '../../../../components/Form/Select/Select';
+
+import shared from '../../../../styleguide/shared.css';
+import m from '../../../../globals/modifiers.css';
+
+export default class Inputs extends Component {
+  state = {
+    checkboxes: [],
+    radios: '',
+    input: '',
```

Comment:
Is there a reason all the inputs use the same value?

Reply:
Is there any reason for them not to use the same value? I intentionally did it to reduce the amount of boilerplate I had to write to make each of the inputs work. Open to other ideas though.

Reply:
It  is probably just me, but I found it distracting seeing other inputs change when typing into the example I was currently reading.

Reply:
It's a tough one—the documentation component is currently 417 lines in length and we're talking about adding at least another six `handleChange` methods and values into the state. We can probably DRY it up... perhaps something like this?

```jsx
handleInputChange = (e, name, value) => {
  this.setState({ [name]: value });
};

render = () => <Input value={ this.state.input } name="input" />;
```

Reply:
Something like the example above would awesome.